java/test/src/main/java/io/ray/test/ObjectRefTransferTest.java

```diff
@@ -0,0 +1,59 @@
+package io.ray.test;
+
+import io.ray.api.ActorHandle;
+import io.ray.api.ObjectRef;
+import io.ray.api.Ray;
+import java.util.ArrayList;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class ObjectRefTransferTest extends BaseTest {
+
+  @BeforeClass
+  public void setUpClass() {
+    System.setProperty("ray.raylet.config.num_workers_per_process_java", "1");
```

Comment:
This is not needed any more. We've change the default value to 1.

Reply:
updated